src/Token.env

```diff
@@ -0,0 +1 @@
+export GITHUB_TOKEN="42386d06a73e346213babd96648992afe3bc52be"
```

Comment:
You're going to want to deactivate this token, make a new one, and add this file to your `.gitignore` so you don't check it in.